x/distribution/types/codec.go

```diff
@@ -9,6 +9,8 @@ func RegisterCodec(cdc *codec.Codec) {
 	cdc.RegisterConcrete(MsgWithdrawDelegatorReward{}, "cosmos-sdk/MsgWithdrawDelegationReward", nil)
 	cdc.RegisterConcrete(MsgWithdrawValidatorCommission{}, "cosmos-sdk/MsgWithdrawValidatorCommission", nil)
 	cdc.RegisterConcrete(MsgSetWithdrawAddress{}, "cosmos-sdk/MsgModifyWithdrawAddress", nil)
+	cdc.RegisterConcrete(QueryDelegatorTotalRewardsResponse{}, "cosmos-sdk/QueryDelegatorTotalRewardsResponse", nil)
```

Comment:
Why do we need to register these? Codecs are only needed for messages iirc.

Reply:
Fixed